www/class/centreon-knowledge/wikiApi.class.php

```diff
@@ -71,12 +71,10 @@ public function __construct()
     private function getCurl()
     {
         $curl = curl_init();
-        $cookiefile = tempnam("/tmp", "CURLCOOKIE");
         curl_setopt($curl, CURLOPT_URL, $this->url);
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($curl, CURLOPT_POST, true);
-        curl_setopt($curl, CURLOPT_COOKIEFILE, $cookiefile);
-        curl_setopt($curl, CURLOPT_COOKIEJAR, $cookiefile);
+        curl_setopt($curl, CURLOPT_COOKIEFILE, '');
```

Comment:
why keep the CookieFile option, when it's empty ? shouldn't it be removed ?

Reply:
Empty means: cookies are in memory